azure-cli.js

```diff
@@ -0,0 +1,51 @@
+const { exec, tryParseAzureOutput } = require("./helpers");
+const { AZURE_LOGIN_COMMAND, DOCKER_IMAGE } = require("./consts.json");
+
+function createDockerCommand(userCommand) {
+  return `
+    docker run \
+    -e AZURE_SERVICE_PRINCIPAL_ID \
+    -e AZURE_SERVICE_PRINCIPAL_CREDENTIAL \
+    -e AZURE_TENANT_ID \
+    -e COMMAND \
+    --rm \
+    ${DOCKER_IMAGE} \
+    ${AZURE_LOGIN_COMMAND} > /dev/null && ${userCommand}
+  `.trim();
+}
+
+function createAzureCommand(userInput) {
```

Comment:
If we called the result, would it really work? Would that be an `azure` command?